atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -425,10 +429,16 @@ def get_build(self, metadata, worker_metadatas):
 
             self.set_go_metadata(extra)
             self.set_group_manifest_info(extra, worker_metadatas)
+            extra['osbs_build']['kind'] = KOJI_KIND_IMAGE_BUILD
+            extra['osbs_build']['engine'] = self.workflow.builder.tasker.build_method
+            if 'operator_manifests' in extra:
+                extra['osbs_build']['subtypes'].append(KOJI_SUBTYPE_OP_APPREGISTRY)
         else:
             source_result = self.workflow.prebuild_results[PLUGIN_FETCH_SOURCES_KEY]
             extra['image']['sources_for_nvr'] = source_result['sources_for_nvr']
             extra['image']['sources_signing_intent'] = source_result['signing_intent']
+            extra['osbs_build']['kind'] = KOJI_KIND_IMAGE_SOURCE_BUILD
+            extra['osbs_build']['engine'] = 'bsi'
```

Comment:
'bsi' should probably be a constant here and the unit test.

Reply:
updated